packages/botkit/src/css/botkit.css

```
@@ -0,0 +1,880 @@
+ * content-hashed path.  Do not hand-edit the generated module.
+ *
+ * All BotKit-owned selectors are namespaced with `bk-` so they never collide
+ * with user-authored post/summary markup, which is styled only in `.bk-prose`.
+ */
+
+/* ------------------------------------------------------------------ *
+ *  Font (bundled locally; served next to this stylesheet)
+ * ------------------------------------------------------------------ */
+
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)